package.json

```diff
@@ -195,7 +201,18 @@
   },
   "targets": {
     "web": {
-      "context": "browser"
+      "context": "browser",
+      "distDir": "./build"
+    },
+    "ext-dev": {
+      "distDir": "./build-ext",
+      "sourceMap": {
+        "inline": true,
+        "inlineSources": true
```

Comment:
This is recommended by https://parceljs.org/recipes/web-extension/#hmr.
I had this comment workaround in proof of concept: https://github.com/oasisprotocol/wallet-poc/blob/redux-ext-poc/package.json#L13-L19